### PR TITLE
module-4-Added ansible directory

### PR DESCRIPTION
**Pull request by RogVan**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

**Comment by BogRad97** [comment]
Your playbook looks good, but you're missing a few things from the submission checklist:
* Inventory file (you committed an empty one) 
* The `.env.j2` file. This should have placeholder values when you add it to git, but the right keys
* You also need to update the Readme file with instructions on how to provision using ansible

**Comment by BogRad97** [comment]
Why did you remove the `.env` and `.env.template` files? They are still needed, `.env.j2` is only used for ansible but we will use the others for everything else

**Comment by RogVan** [comment]
Yeah I realised that. I've branched module 5 off module 4 and re-added the .env and .env.template files back in there so should be fine

File: templates/.env.j2 [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+# Change the following values for local development.
+SECRET_KEY=secret-key
+
+# Trello Authentication
+TRELLO_KEY={{ trello_key }}
+TRELLO_TOKEN={{ trello_token }}
+
+#Trello Ids
+TRELLO_BOARD_ID='63039c2f125ba22bbd3db407'
```
Comment (BogRad97) [inline templates/.env.j2:13]: Even though these are not very dangerous to leave in the git history, they are still arguably secrets. I would probably replace these with placeholder values, and add them in the playbook as variables.

**Review by BogRad97 — APPROVED** [review]
Looks good to me :)

**Pull request merged**